packages/easy-email-extensions/src/ShortcutToolbar/components/BlocksPanel/presetTemplate/index.tsx

```diff
@@ -19,140 +19,118 @@ import { ColumnBlockItem } from './ColumnBlockItem';
+        title: t('Accordion'),
+        description:t('accordionDescriptionText'),
         component: AccordionBlockItem,
       },
       {
         type: AdvancedType.CAROUSEL,
-        title: 'Carousel',
-        description: `This block displays a gallery of images or "carousel".
-        Readers can interact by hovering and clicking on
-        thumbnails depending on the email client they use.`,
+        title: t('Carousel'),
+        description: t('carouselDescriptionText'),
         component: CarouselBlockItem,
       },
       {
         type: AdvancedType.SOCIAL,
-        title: 'Social',
-        description: `Displays calls-to-action for various social networks with
-        their associated logo.`,
+        title: t('Social'),
+        description: t('socialDescriptionText'),
         component: SocialBlockItem,
       },
     ],
   },
   {
-    title: 'Layout',
+    title: t('Layout'),
     name: 'LAYOUT',
     blocks: [
       {
         type: AdvancedType.WRAPPER,
-        title: 'Wrapper',
-        description: `Wrapper enables to wrap multiple sections together. It's especially useful to achieve nested layouts with shared border or background images across sections.
-        `,
+        title: t('Wrapper'),
+        description: t("wrapperDescriptionText"),
         component: WrapperBlockItem,
       },
       {
         type: AdvancedType.SECTION,
-        title: 'Section',
+        title: t('Section'),
         description: (
           <Stack vertical spacing='none'>
             <TextStyle>
-              Sections are intended to be used as rows within your email. They
-              will be used to structure the layout.
+              {t('sectionDescriptionText1')}
             </TextStyle>
             <TextStyle>
-              Sections cannot nest in sections. Columns can nest in sections;
-              all content must be in a column.
+             {t('sectionDescriptionText2')}
             </TextStyle>
           </Stack>
         ),
         component: SectionBlockItem,
       },
       {
         type: AdvancedType.GROUP,
-        title: 'Group',
-        description: `Group allows you to prevent columns from stacking on
-          mobile. To do so, wrap the columns inside a group
-          block, so they'll stay side by side on mobile.`,
+        title: t('Group'),
+        description: t('groupDescriptionText'),
         component: GroupBlockItem,
       },
       {
         type: AdvancedType.COLUMN,
-        title: 'Column',
+        title: t('Column'),
         description: (
           <Stack vertical spacing='none'>
             <TextStyle>
-              Columns enable you to horizontally organize the content within
-              your sections. They must be located under "Section" block in order
-              to be considered by the engine. To be responsive, columns are
-              expressed in terms of percentage.
+              {t(`columnDescriptionText1`)}
```

Comment:
```suggestion
              {t(`Columns enable you to horizontally organize the content within
              your sections. They must be located under "Section" block in order
              to be considered by the engine. To be responsive, columns are
              expressed in terms of percentage.`)}
```

Reply:
In my opinion keywords are more usefull for localization files. i changed keyword naming 2 to 1 in json files, according to your comment. But didnt changed tsx file.

Reply:
If you still insist, I will change it as you say.

Reply:
https://github.com/zalify/easy-email/blob/master/scripts/translate.ts#L14-L45

Pleaase check this script.
We keep all the text as the keywords for script translation, while there is no corresponding translation, the content of the text can still be displayed.

Reply:
If you have google serverice google, you can run this script， just need to run 
 
 ```bash
 npm run translate
 ```